addons/bestja_frontend_ucw/static/src/css/custom.css

```diff
@@ -21,6 +21,50 @@ a:hover, a:focus{
     text-decoration: none;
 }
 
+.field-join-info h4{
+    font-size: 14px;
+    color: #737373;
+}
+
+.field-join-info span{
+    font-size: 13px;
+}
+
+.field-join-info span:hover:after{
+    content: "Jeżeli chcesz zostać koordynatorem w swojej organizacji, wypełnij formularz, aby w kolejnym kroku uzupełnić dane swojej organizacji.";
```

Comment:
This looks like a bit too long and significant text to be put in a CSS class. Also, this solution would be problematic if we used translations. Maybe we should put this content in a separate container with `display: none` and display it using `.field-join-info span:hover .our-container` selector?